data/migrations/34.lua

```diff
@@ -38,30 +21,27 @@ function onUpdateDatabase()
 		result.free(resultId)
 	end
 
-	local resultId = db.storeQuery(string.format("SELECT `player_id`, `key`, `value` FROM `player_storage` WHERE `key` >= %d AND `key` <= %d", mountRange, mountRange + 10))
-	if resultId then
-		repeat
-			local playerId = result.getNumber(resultId, "player_id")
-			local key = result.getNumber(resultId, "key")
-			local mounts = {}
-
-			for i = 1, 300 do
-				if key == mountRange + (i % 31) then
-					table.insert(mounts, i-1)
-				end
-			end
-
-			for k, v in pairs(mounts) do
-				local lshift = bit.lshift(1, (v % 31))
+	-- this equals to our current id range from mounts
+	for i = 1, 200 do
+		local key = mountRange + ((i-1) / 31)
+		local resultId = db.storeQuery(string.format("SELECT `player_id`, `key`, `value` FROM `player_storage` WHERE `key` = %d", key))
+		if resultId then
+			repeat
+				local playerId = result.getNumber(resultId, "player_id")
+				local lshift = bit.lshift(1, ((i-1) % 31))
 				local mount = bit.band(lshift, result.getNumber(resultId, "value"))
-				if mount then
-					db.query(string.format("INSERT INTO `player_mounts` (`player_id`, `mount_id`) VALUES (%d, %d)", playerId, v+1))
+
+				if mount ~= 0 then
+					db.query(string.format("INSERT INTO `player_mounts` (`player_id`, `mount_id`) VALUES (%d, %d)", playerId, i))
 				end
-			end
+			until not result.next(resultId)
+			result.free(resultId)
+		end
+	end
 
-			db.asyncQuery(string.format("DELETE FROM `player_storage` WHERE `player_id` = %d AND `key` = %d", playerId, key))			
-		until not result.next(resultId)
-		result.free(resultId)
+	for i = 0, 9 do
+		db.asyncQuery(string.format("DELETE FROM `player_storage` WHERE `key` = %d", mountRange + i))
```

Comment:
Again, just use >= and <=

Reply:
nice, should have thought of that idea, thanks!